public/css/styles.css

```diff
@@ -0,0 +1,673 @@
+}
+
+.column-two {
+  -webkit-column-count: 2;
+     -moz-column-count: 2;
+     -ms-column-count: 2;
+          column-count: 2;
+}
+.column-right {
+  float: right;
+}
+.section-orange {
+  background-color: #FF6124;
+  color: #fff;
+  padding-bottom: 2em;
+}
+
+.section-black {
+  background-color: #585050;
+  color: #fff;
+}
+
+/*---Navigation menu css---*/
+.hidden-class {
+  display: inline-block;
+}
+
+.nav-li {
+  margin-left: 3em;
+}
+
+.drop-down {
+  font-size: 1.2em;
+}
+
+/*---main header css---*/
+.header-section {
+  font-size: 2em;
+  font-family: helvetica;
+  height: 17em;
+}
+
+.header-section {
+  background-image: url("../imgs/top-bkgrnd.jpg");
+}
+
+/*---services section css---*/
+.flex-container {
+  width: 100%;
+  display: flex;
+  display: -webkit-flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: center;
+  align-items: flex-start;
+  -webkit-justify-content: center;
+  -webkit-flex-direction: row;
+  -webkit-flex-wrap: wrap;
+}
+
+ .flex-item-1 {
+  width: 100%;
+  height: 24em;
+  text-align: center;
+}
+
+.flex-item-2 {
+  width: 100%;
+  height: 20em;
+  text-align: center;
+  margin-left: -.5em;
+}
+.flex-item-3 {
+  width: 22%;
+  height: 20em;
+  text-align: center;
+}
+.flex-item-4 {
+  width: 22.5%;
+  text-align: center;
+}
+
+.table-section {
+  width: 100%;
+  display: flex;
+  display: -webkit-flex;
+  justify-content: center;
+  padding-bottom: 3em;
+  padding-right: 5%;
+  padding-left: 5%;
+  -webkit-justify-content: center;
+}
+
+table {
+  position: relative;
+  float: left;
+  margin-right: 1em;
+  font-size: .8em;
+  margin-bottom: 2em;
+}
+th {
+  background-color: #703A20;
+  color: #fff;
+}
+tr {
+  background-color: #fff;
+  color: #000;
+}
+tr:not(.table-head) {
+  border-bottom: .1em solid #FF6124;
+}
+td {
+  padding-top: .5em;
+  padding-bottom: .5em;
+}
+th, .table-price {
+  padding: 1em;
+}
+.table-price {
+  color: #FF6124;
+}
+.table-foot {
+  background-color: #C1530D;
+  color: #fff;
+}
+.button-signup {
+  background-color: #FF6124;
+  margin: 1em;
+  border: none;
+  padding-left: 1em;
+  padding-right: 1em;
+}
+.section-orange p, .section-black p {
+  line-height: 1.5em;
+}
+
+/*---About and skills section css---*/
+progress {
+  width: 83%;
+  background-color: #FF6124;
+}
+progress::-moz-progress-bar {
+  background: #FF6124;
+}
+progress::-webkit-progress-value {
+  background-color: #FF6124;
+}
+.skill-list {
+  font-size: .8em;
+  text-align: left;
+  margin-left: 8.5em;
+  margin-bottom: auto;
+}
+.skills-para {
+  padding: 2em 0;
+}
+
+/*---portfolio section css---*/
+nav a {
+  text-decoration: none;
+  color: #686868;
+  font-weight: bold;
+  font-size: 1.15em;
+}
+nav ul {
+  margin-left: -2em;
+}
+
+nav a:hover {
+  color: #000000;
+}
+
+.portfolio-nav a:hover{
+  background-color: #FF6124;
+  color: #ffffff;
+}
+
+.portfolio-nav li{
+  margin-right: 1em;
+}
+
+.carousel-lrg {
+  width: 66.5em;
+}
+
+/*---Team and clients CSS---*/
```

Comment:
Overall, the [leading](https://creativemarket.com/blog/2014/09/18/whats-the-difference-between-leading-kerning-and-tracking) on this section of the page is very tight. Your designer _will_ notice things like that, so you should learn to recognize it as well.